tst/functions/Mock.Tests.ps1

```diff
@@ -1287,9 +1287,6 @@ Describe 'Dot Source Test' {
     }
 
     It "Doesn't call the mock with any other parameters" {
-        InPesterModuleScope {
```

Comment:
didn't make any sense and was only reference to `$script:mocktable` in `Main.ps1`